src/components/Card/itemCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const card ={
+  cursor: 'pointer',
+};
+
+const Card = ({index, setName, setPrice, onClick,}) => {
+    return (
+      <div           
+        onClick={onClick}
+        className= 'Card'
+        >
+          <img className = 'Card__img' src='./images/curtains.jpeg' alt='curtains' />
+          <h1 className = 'Card__indexName'>{`${index} ${setName}`}</h1>
```

Comment:
One small change is needed, in Bem we name things not "indexName", but "index-name" (and do we really need to call it index?)))))